core/modules/layout/layout.module

```diff
@@ -802,6 +802,7 @@ function layout_theme() {
           'content' => NULL,
           'settings' => NULL,
           'layout' => NULL,
+          // @todo: Rename to layout_template_info in 2.x.
```

Comment:
Would it be worth adding `layout_template_info` and essentially deprecating (but leaving functional) `layout_info` now, so people can start using the new/correct variable?